lib/dry/types/enum.rb

```diff
@@ -26,6 +26,10 @@ def call(input)
         type[value || input]
       end
       alias_method :[], :call
+
+      def include?(input)
+        values.include?(input)
+      end
     end
```

Comment:
Would `alias_method :include?, :valid?` do the trick?

Reply:
I tried and it will do the trick @solnic @d-Pixie.

The `valid?` method is `Decorator` file if you want to have a look.

https://github.com/dry-rb/dry-types/blob/master/lib/dry/types/decorator.rb#L27